openedx/core/djangoapps/user_authn/views/login_form.py

```diff
@@ -15,6 +15,7 @@
 
 import third_party_auth
 from edxmako.shortcuts import render_to_response
+from openedx.adg.common.user_authn_override.forms import RegistrationFormFactory
```

Comment:
I guess we should revert it now.

Reply:
No, We need this new form for the current field customizations, Our new form fields will appear at the end of this form.